Rudder/Classes/RSConfigBuilder.m

```diff
@@ -17,7 +17,7 @@ - (instancetype) withEndPointUrl:(NSString *)endPointUrl{
         config = [[RSConfig alloc] init];
     }
     NSURL *url = [[NSURL alloc] initWithString:endPointUrl];
-    config.dataPlaneUrl = [[NSString alloc] initWithFormat:@"%@://%@", [url scheme], [url host]];
+    config.dataPlaneUrl = [[NSString alloc] initWithFormat:@"%@://%@:%@", [url scheme], [url host], [url port]];
```

Comment:
What if the dataPlaneUrl does not contain any port? Wouldn't we be adding an extra `:` then?

Reply:
checked